### DISCUSSION TITLE
S&P 500 Components Correction

### DISCUSSION THREAD

**Discussion by rkkc2003** [fja05680/sp500#7] [Q&A]
KORS changed its name to CPRI on 2 Jan 2019 https://www.marketwatch.com/story/michael-kors-to-change-name-to-capri-holdings-on-jan-2-2018-12-31 
The ... Components & Changes (10-18-2021).cvs should be corrected:
1. CPRI shouldn't be listed until 2 Jan 2019 (currently, it is listed since 4 Jan 2016 in the 10-18-2021 file)
2. KORS should be listed until 31 Dec 2018 (it is unlisted since 18 Sep 2018 in the 10-18-2021 file)

**Comment by fja05680** [comment]
Thanks for bring this to my attention.  It is a problem in the original cvs file from "Trading Evolved".  For this type of problem, I'll write some code to read, then patch that file.  I'll plan to take care of this by the time I update again.

**Comment by mstollenwerk** [comment]
Hello Farrell,
Thank you very much for your work! I also have a potential correction:
Ticker RE changed its name to ticker EG on July 10, 2023. In your data it is JUNE 10, 2023. 
Best regards,
Michael

**Reply by mstollenwerk** [reply]
Similarly WLTW changed to WTW on Jan 10, 2022. In your data its Jan **20**, 2022.

**Reply by fja05680** [reply]
Michael, thanks for pointing these out.  I have corrected them.
